dc-infrastructure.tf

```diff
@@ -11,6 +11,7 @@ module "base-infrastructure" {
   domain               = var.domain
   namespace            = local.namespace
   eks_additional_roles = var.eks_additional_roles
+  whitelist_cidr       = var.whitelist_cidr
```

Comment:
https://hello.atlassian.net/wiki/spaces/CONTENT/pages/542809951/Terms+Whitelist+Blacklist+in+software+Allowlist+Blocklist